### ISSUE TITLE
  HDF5. File accessibility. Unable to open file.

### ISSUE THREAD

**Issue by Agsguo** [js2264/HiCExperiment#11]
My COOL file is intact, but it says “error H5Fopen(file, flags = flags, fapl = fapl, native = native): 
  HDF5. File accessibility. Unable to open file.”
![1722667669255](https://github.com/user-attachments/assets/e280b092-63dd-4539-ab95-31f5bf2f3f26)
![1722667704352](https://github.com/user-attachments/assets/3cb2070b-8bc6-4a97-bcb8-b65a21286b56)

**Comment by js2264** [comment]
Can you provide the cool file so I can troubleshoot? This suggests that the cool file may be malformed
